cranelift/filetests/filetests/isa/x86/simd-arithmetic-run.clif

```diff
@@ -192,31 +192,22 @@ block0:
 }
 ; run
 
-function %fmax_f64x2() -> b1 {
-block0:
-    v0 = vconst.f64x2 [-0.0 -0x1.0]
-    v1 = vconst.f64x2 [+0.0 +0x1.0]
-
+function %fmax_f64x2(f64x2, f64x2) -> f64x2 {
+block0(v0: f64x2, v1: f64x2):
     v2 = fmax v0, v1
-    v3 = fcmp eq v2, v1
-    v4 = vall_true v3
-
-    return v4
+    return v2
 }
-; run
-
-function %fmin_f64x2() -> b1 {
-block0:
-    v0 = vconst.f64x2 [-0x1.0 -0x1.0]
-    v1 = vconst.f64x2 [+0.0 +0x1.0]
+; run: %fmax_f64x2([-0x0.0 -0x1.0], [+0x0.0 0x1.0]) == [+0x0.0 0x1.0]
+; run: %fmax_f64x2([-NaN NaN], [0x0.0 0x100.0]) == [-NaN NaN]
```

Comment:
Can you add a few more tests, please:

- a test with only one NaN in one lane of one input (should return cNaN for this lane)
- a test with an arithmetic (non-canonical) NaN in one lane of one input (should cNaN too), hopefully our parser supports this :)
- a test with -NaN in one lane of one vector, and +1 in the same lane of the other vector?